app/routes/paymentdetail.js

```diff
@@ -15,14 +15,14 @@ module.exports = {
         part_postcode: records[0].part_postcode,
         town: records[0].town,
         county_council: records[0].county_council,
-        financial_year: records[0].financial_year,
         parliamentary_constituency: records[0].parliamentary_constituency,
         schemes: records.map(r1 => {
           return {
-            scheme: r1.scheme,
+            name: r1.scheme,
```

Comment:
Since 'schemes' represent a list of scheme objects, doesn't that automatically infer whose name its referring to? For example, if you had a Employee table in the db, you would have a field called 'Name' rather than 'Employee Name'.

Reply:
I dont think rest of fields follows that way here,  example scheme_detail. Hence I thought we can leave as it is , as the existing name stays more closer. otherwise all other fields need to be changed uniformly.

Reply:
I have changed the scheme_detail as well now.